src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import core.basesyntax.service.GenerateReport;
+import core.basesyntax.service.ReadDataFile;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        String inputFileName = "src/main/resources/InputFile.csv";
+        String resultFileName = "src/main/resources/EndDayReport.csv";
```

Comment:
Avoid hardcoding file paths within classes. These should be configurable and ideally set in the `Main` class.